app/helpers/setTheme.js

```diff
@@ -0,0 +1,14 @@
+import { helper } from '@ember/component/helper';
+import { THEME } from '../constants/theme';
+
+function setTheme(theme) {
+  if (theme === THEME.LIGHT) {
+    document.body.classList.add('lightTheme');
+    document.body.classList.remove('darkTheme');
+  } else {
+    document.body.classList.remove('lightTheme');
+    document.body.classList.add('darkTheme');
+  }
```

Comment:
Did we try `toggling` classes here?

Reply:
https://github.com/Real-Dev-Squad/website-my/pull/160#discussion_r748691248
Yes. If a user is on dark mode and user reloads the page, then classes will get toggled due to which he will see light mode, to overcome this we have to write extra code for handling this. But by using `add` and `remove` we are handling it efficiently.